src/test/java/uk/gov/companieshouse/extensions/api/requests/MongoDBTest.java

```diff
@@ -0,0 +1,145 @@
+package uk.gov.companieshouse.extensions.api.requests;
+
+import com.mongodb.MongoClient;
+
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import org.bson.Document;
+import org.bson.json.JsonWriterSettings;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.companieshouse.extensions.api.groups.Integration;
+import uk.gov.companieshouse.extensions.api.reasons.ExtensionReasonEntity;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+
+import static com.mongodb.client.model.Filters.eq;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests require actual connection to mongo db in order to run so these will be ignored
+ * so that testing does not fail on concourse just because this connection is absent.
+ */
+
+
+@Category(Integration.class)
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class MongoDBTest {
+
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'");
+    public static final String WINTER_TEST_DOCUMENT = "winter_systime_test_123";
+    public static final String SUMMER_TEST_DOCUMENT = "summer_systime_test_123";
+
+    @Autowired
+    private ExtensionRequestsRepository requestsRepository;
+
+    private MongoClient mongoClient = new MongoClient();
+
+    private LocalDate winterStart;
+    private LocalDate winterEnd;
+    private LocalDate summerStart;
+    private LocalDate summerEnd;
+
+    @Value("${TEST_MONGO_DB_URL}")
+    private String testMongoUrl;
+
+    @Before
+    public void init() {
+        winterStart = LocalDate.of(2020,1,1);
+        winterEnd = LocalDate.of(2020,1,6);
+        summerStart = LocalDate.of(2020,7,1);
+        summerEnd = LocalDate.of(2020,7,6);
+
+        MongoClientURI connectionString = new MongoClientURI(testMongoUrl);
+        mongoClient = new MongoClient(connectionString);
+    }
+
+    @After
+    public void removeTestDocuments() {
+        requestsRepository.deleteById(WINTER_TEST_DOCUMENT);
+        requestsRepository.deleteById(SUMMER_TEST_DOCUMENT);
+    }
+
+    @Ignore
```

Comment:
Minor, but Ignore annotation could move to the class level since all tests are being ignored. It then also 'lives' closer to the comment in the class Javadoc which explains why they are ignored. Up to you. Approving.